src/easylink/steps/dev/python_pandas/python_pandas.def

```diff
@@ -16,18 +17,6 @@ From: python@sha256:1c26c25390307b64e8ff73e7edf34b4fbeac59d41da41c08da28dc316a72
 
 %environment
     export LC_ALL=C
-    export PYTHONPATH=/app:$PYTHONPATH
```

Comment:
These changes to the dummy pipeline implementation scripts are just the result of what we found when creating the implementation-creator tool (i.e. we don't need labels or a startscript)